bundles/org.openhab.binding.pushbullet/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -6,7 +6,7 @@
 
 	<thing-type id="bot">
 		<label>Pushbullet Bot</label>
-		<description>Bot to send messages with.</description>
+		<description>Bot to send messages with</description>
```

Comment:
Don't we usually consider descriptions as full sentences and end with punctuation?